cmd/minikube/cmd/cp.go

```diff
@@ -59,6 +59,7 @@ var cpCmd = &cobra.Command{
 			out.ErrLn("%v", errors.Wrap(err, "getting file asset"))
 			os.Exit(1)
 		}
+		defer fa.Close()
```

Comment:
in all of these defers please log the error

for example
```

	defer func() {
      err := fa.Close()
if err !=nil {
       klog.Warningf("error closing the file %s: %v",fa.Name(), err)
}
}
```

Reply:
change to logging when close failed.